### ISSUE TITLE
Last Used Category is not filled in when selecting Payee from filtered list

### ISSUE THREAD

**Issue by tactilis** [moneymanagerex/moneymanagerex#4836]
MMEX version:
 - [x] 1.5.17
 
Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

From https://forum.moneymanagerex.org/viewtopic.php?p=23105

If the Option: Default Deposit/Withdrawal Category is set to 'Last used for payee' and you create a new transaction, then:

1. If you start typing in the Payee field and select the payee from the filtered list, the last used category is _not_ filled in. [Bug]

However:

2. If you click on the Payee combobox dropdown arrow and select the payee, then the last used category _is_ filled in. [Correct behaviour]

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Start typing in the Payee field and select the payee from the filtered list

Expected result:

The last used category for that payee is filled in

Actual result:

The category field stays empty.

**Comment by rmelillo76** [comment]
Thanks @tactilis I have noticed this as well and was going to create an issue but you beat me to it! :)  It seems there must be a different change event when typing and autocomplete is used. I am going to create a separate issue because pressing enter to display the Organize Payees dialog populates the selected payee but not the category either. Different flow, but similar outcome as this item.

**Comment by whalley** [comment]
I can't reproduce this on MacOS. 

@vomikan Can you investigate?

**Comment by vomikan** [comment]
> I can't reproduce this on MacOS.
> 
> @vomikan Can you investigate?

It seems outdated.
